src/components/editor/property/PropertyComponent.jsx

```diff
@@ -10,6 +10,19 @@ import { getTagNameForPropertyTemplate } from 'utilities/propertyTemplates'
 
 const PropertyComponent = (props) => {
   const tag = getTagNameForPropertyTemplate(props.propertyTemplate)
+  if (!tag) {
+    return (
+      <div className="row">
+        <div className="col-md-12" style={{ marginTop: '10px' }}>
+          <div className="alert alert-danger alert-dismissible">
+            <button className="close" data-dismiss="alert" aria-label="close">&times;</button>
```

Comment:
Should the error be dismissible? Could clicking close here for alert give the false impression that the error is in someway addressed?

Reply:
If you don't think it is a problem; just resolve this conversation and I'll approve so we can merge the PR

Reply:
mebbe we can address with follow-on issue/PR?  This is certainly an improvement over former behavior ...

Reply:
Also - we really would like to get folks using resourceTemplates that adhere to 0.2.0 schemas.  So not sure how often we'll hit this?

Another approach would be to have it display at the top of the screen, via ErrorMessages component.

This seemed potentially more useful but 🤷‍♀ .

I hope it's ok if I leave the ticket making to you, as I'm not really sure what the desired behavior would be.